tests/base/pom.xml

```diff
@@ -642,11 +642,6 @@
             <artifactId>apacheds-kerberos-codec</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.directory.server</groupId>
-            <artifactId>apacheds-protocol-kerberos</artifactId>
```

Comment:
Can this be removed from the main pom.xml as well?

Reply:
done